api/proto/info/info.proto

```diff
@@ -419,6 +419,21 @@ enum ZDeviceState {
     ZDEVICE_STATE_POWERING_OFF = 7;
 }
 
+// Information about snapshot in zfs for zvol
+message ZfsSnapshotInfo {
```

Comment:
It is expected to have state inside info structure to see what happen with the object.

Reply:
@giggsoff What state for snapshot do you mean? 
Current snapshot operations are quite fast

Reply:
Can we possible have errors (error state)? Deleted is another state.